registry.json

```diff
@@ -12,6 +12,20 @@
     "telegram": "biopool",                                          
     "operatorName": "Beatrix",                                   
     "github": "BioPool" 
+ },
+  {
+    "ticker": "Lotto",                                                       
+    "poolId": "ff37a9645f9d5ab34192475136bf5ceb1c18166bb40a3b1637f4022b", 
+    "social": {       
+      "website": "",
+      "twitter": "LottoLabs", 
+      "instagram": "",
+      "youtube": "",
+      "facebook": ""
+    },
+    "telegram": "",                                          
```

Comment:
You must include a telegram username and it must be the username of an individual and not a group or channel

Reply:
Updated, thanks missed that.